### PR TITLE
fix: consistent use of force_delete in kms modules

### PR DESCRIPTION
**Pull request by iamar7**
### Description

https://github.ibm.com/GoldenEye/issues/issues/9228

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: examples/basic/variables.tf [hunk 1]
```diff
@@ -13,3 +13,9 @@ variable "existing_kms_instance_guid" {
   type        = string
   description = "GUID of an existing kms instance"
 }
+
+variable "force_delete" {
```
Comment (ocofaigh) [inline examples/basic/variables.tf:17]: don't expose this in the example - hard code it to `true` and leave a code comment saying its set to true for testing purposes

File: examples/complete/variables.tf [hunk 2]
```diff
@@ -25,3 +25,9 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "force_delete" {
```
Comment (ocofaigh) [inline examples/complete/variables.tf:29]: don't expose this in the example - hard code it to `true` and leave a code comment saying its set to true for testing purposes

**Comment by iamar7** [comment]
Need to skip the upgrade test - 

Screenshot 2024-05-24 at 3 17 28 PM

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**